services/notify/src/exchanges.js

```diff
@@ -75,3 +75,18 @@ exchanges.declare({
   routingKeyBuilder: commonRoutingKeyBuilder,
   CCBuilder: commonCCBuilder,
 });
+
+exchanges.declare({
```

Comment:
Do you mean `sqs` if message is for `user` or else use `pulse` if message is for `channel`. But then it won't be cloud-agnostic! We can have some scope like thing so that anyone can't just bind to that exchange `irc-notification` ?

Reply:
Very little is secret in Taskcluster.  The task that generated the notification isn't secret.  And irc isn't secret.  I'm not too worried about this.  A note in the docs might help :)

"Note that all notifications are transmitted via pulse and should not be considered private."